api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -114,6 +129,18 @@ message RBAC {
   // Maps from policy name to policy. A match occurs when at least one policy matches the request.
   // The policies are evaluated in lexicographic order of the policy name.
   map<string, Policy> policies = 2;
+
+  // Condition for the audit logging to be happen.
+  // If this condition is met, all the audit loggers configured here will be invoked.
+  //
+  // [#not-implemented-hide:]
+  AuditCondition audit_condition = 3;
+
+  // Configurations for RBAC-based authorization audit loggers.
+  //
+  // [#not-implemented-hide:]
+  // [#extension-category: envoy.rbac.audit_loggers]
+  repeated core.v3.TypedExtensionConfig audit_log = 4;
```

Comment:
Logically I think it would make more sense to have a new message called `AuditLog` and then put these fields within that message. That way all of the fields can be grouped together or not.

On this field I would use PGV to specify at least one logger configured. And on the enum I would require a valid value.

Reply:
Thank you, @mattklein123, for the quick review! I wrapped them and added the PGVs. To avoid identical field names, I renamed the wrapping message `AuditOptions`. Please let me know what you think. I don't have strong opinions so could change it back to `AuditLog` if preferred.